src/Template/Pats/Pats.php

```diff
@@ -13,47 +13,66 @@
 
 class Pats
 {
-    const SIZE = 8;
-
-    const PAT1 = <<<EOT
+    const GOOD_JOB = <<<EOT
 Good job @{{ author }}.
 EOT;
 
-    const PAT2 = <<<EOT
+    const YOU_WERE_FAST = <<<EOT
 You were fast on this one, thanks @{{ author }}.
 EOT;
 
-    const PAT3 = <<<EOT
+    const PATCH = <<<EOT
 Good catch @{{ author }}, thanks for the patch.
 EOT;
 
-    const PAT4 = <<<EOT
+    const THANK_YOU = <<<EOT
 Thank you @{{ author }}.
 EOT;
 
-    const PAT5 = <<<EOT
+    const GOOD_CATCH_THANKS = <<<EOT
 Good catch, thanks @{{ author }}.
 EOT;
 
-    const PAT6 = <<<EOT
+    const THANKS_FOR_PR = <<<EOT
 Thanks @{{ author }} for the pull request!
 EOT;
 
-    const PAT7 = <<<EOT
+    const WELL_DONE = <<<EOT
 Well done @{{ author }}.
 EOT;
 
-    const PAT8 = <<<EOT
+    const BEERS = <<<EOT
 :beers: @{{ author }}.
 EOT;
 
+    public static function getDefaultPats()
+    {
+        static $pats = [];
+
+        if (!$pats) {
+            $r = new \ReflectionClass(get_class());
+            $pats = array_flip($r->getConstants());
+            array_walk($pats, function(&$value) {
```

Comment:
@sstok no, I need to keep the array keys, which in this point are the real values (note the 2 calls to `array_flip()`).